### PR TITLE
add periodic job for live-iso build against ocp 4.8

### PR DESCRIPTION
**Pull request by osherdp**
/hold
/cc @osherdp

### CODE REVIEW

**Comment by omertuc** [comment]
> /cc @osherdp
Did you just cc yourself?

/lgtm

File: ci-operator/config/openshift/release/openshift-release-master__ocp-4.8.yaml [hunk 1]
```diff
@@ -211,6 +211,18 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp-libvirt-cert-rotation
+- artifact_dir: /tmp/artifacts
+  as: e2e-aws-single-node
+  cron: 0 0 * * 1
```
Comment (romfreiman) [inline ci-operator/config/openshift/release/openshift-release-master__ocp-4.8.yaml:216]: is it the same time when all assisted periodic jobs are running? Maybe to split so there will be less tress on the packet servers

File: ci-operator/config/openshift/release/openshift-release-master__ocp-4.8.yaml [hunk 2]
```diff
@@ -211,6 +211,18 @@ tests:
...
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws-single-node
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-single-node-live-iso
+  cron: 0 0 * * 1
```
Comment (romfreiman) [inline ci-operator/config/openshift/release/openshift-release-master__ocp-4.8.yaml:222]: https://github.com/openshift/origin/pull/25902

**Comment by osherdp** [comment]
depends on https://github.com/openshift/assisted-test-infra/pull/559

**Comment by osherdp** [comment]
duplicate: #16447

**Pull request closed without merging**